tools/Customization/DevHome.Customization/CustomControls/DevDriveOptimizerCardBody.xaml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.Customization.CustomControls.DevDriveOptimizerCardBody"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:devEnvConverters="using:DevHome.Common.Environments.Converters"
+    mc:Ignorable="d">
+    <UserControl.Resources>
+        <devEnvConverters:CardStateColorToBrushConverter x:Key="CardStateColorToBrushConverter"/>
+        <devEnvConverters:CardStateToLocalizedTextConverter x:Key="CardStateToLocalizedTextConverter"/>
+    </UserControl.Resources>
+
+    <Grid
+            Style="{StaticResource CardBodyContainerStyle}">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+        <!--
+                Stackpanel containing name of cache to be optimized and a description.
+            -->
+        <StackPanel
+                Grid.Column="1"
+                Grid.ColumnSpan="2"
+                Style="{StaticResource CardBodyStackPanelStyle}">
+            <StackPanel Orientation="Vertical">
+                <Grid Margin="50 10 50 10">
+                    <TextBlock Margin="5" Grid.Column="0"
+                        Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}"
+                        Text="{x:Bind CacheToBeMoved, Mode=OneWay}"
+                        IsTextSelectionEnabled="True"/>
+                    <Grid HorizontalAlignment="Right">
+                        <FontIcon HorizontalAlignment="Left" Margin="-10" Glyph="&#xE946;" FontSize="12" FontFamily="{StaticResource SymbolThemeFontFamily}"/>
+                        <TextBlock HorizontalAlignment="Left" Margin="5 5 5 5" Grid.Column="1"
+                            Foreground="PowderBlue"
+                            Text="Suggestion"
+                            IsTextSelectionEnabled="True"/>
+                    </Grid>
+                </Grid>
+                <Line Margin="-5 10 -5 0" Stroke="Black" Stretch="Fill" X1="0" Y1="0" X2="1" Y2="0" StrokeThickness="1"/>
+                <Grid Margin="50 10 10 10">
+                    <StackPanel>
+                        <TextBlock TextWrapping="Wrap" Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}">                            
+                            <Run>Global environment variable </Run>

```

Comment:
Put this whole sentence in a resource file.

Reply:
We have substrings with different colors. I can't seem to find a way of doing this using the resource file. Lmk if there is a work-around.

Reply:
Look for places in a resw file where you see {0}. That is the proper way to do this. What you have cannot be translated.

Reply:
Yes, I have used {i}s for localization, yet to push the changes. However I am not sure how to have the {i}s show up in different color. For now I will make the change for the entire string to show up with the default color.